.gitignore

```diff
@@ -1,8 +1,7 @@
-#/vendor
+vendor/
 bootstrap/cache/
 storage/
-env
+.env.*.php
```

Comment:
This shouldn't be ignored. 
`.env.blade.php` is the configuration template used by the setup script, which should remain in the Git repo and not ignored.